src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-list/pool-list.component.ts

```diff
@@ -120,60 +125,57 @@ export class PoolListComponent extends ListWithDetails implements OnInit {
         cellTransformation: CellTemplate.executing
       },
       {
-        prop: 'type',
-        name: $localize`Type`,
+        prop: 'data_protection',
+        name: $localize`Data Protection`,
+        cellTransformation: CellTemplate.badge,
+        customTemplateConfig: {
+          class: 'badge-hdd'
```

Comment:
Wouldn't it make sense to introduce a new CSS class here? I guess you simply want to use a color here, but using a class that is not related to this column feels wrong. Maybe simply add
```
.badge-background-gray,
.badge-hdd {
  background-color: $gray-600;
  color: $white;
}
```
```suggestion
          class: 'badge-background-gray'
```
I think this is a better approach in terms of branding, too.